src/App.js

```diff
@@ -3,6 +3,7 @@ import './App.css';
 function App() {
   return (
     <div className="App">
+      <Navbar/>
```

Comment:
Implement Route and Switch to each component

Reply:
Implement Route and Switch to each component